accord-core/src/main/java/accord/primitives/RangeDeps.java

```diff
@@ -18,24 +18,55 @@
 
 package accord.primitives;
 
-import accord.api.Key;
-import accord.api.RoutingKey;
-import accord.utils.*;
-import accord.utils.SortedArrays.SortedArrayList;
-import net.nicoulaj.compilecommand.annotations.DontInline;
-import net.nicoulaj.compilecommand.annotations.Inline;
-
-import javax.annotation.Nonnull;
-import javax.annotation.Nullable;
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.BitSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
 import java.util.function.BiConsumer;
 import java.util.function.Consumer;
 import java.util.function.Function;
 import java.util.function.Predicate;
 import java.util.stream.Stream;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import accord.api.Key;
+import accord.api.RoutingKey;
+import accord.utils.ArrayBuffers;
+import accord.utils.IndexedBiConsumer;
+import accord.utils.IndexedConsumer;
+import accord.utils.IndexedQuadConsumer;
+import accord.utils.IndexedRangeQuadConsumer;
+import accord.utils.Invariants;
+import accord.utils.RelationMultiMap;
+import accord.utils.SearchableRangeList;
+import accord.utils.SortedArrays;
+import accord.utils.SortedArrays.SortedArrayList;
+import accord.utils.SymmetricComparator;
+import accord.utils.TriFunction;
+import net.nicoulaj.compilecommand.annotations.DontInline;
+import net.nicoulaj.compilecommand.annotations.Inline;
 
-import static accord.utils.ArrayBuffers.*;
```

Comment:
silly `.*`!

Reply:
Are you asking to keep the `.*`? Cassandra code style is to use explicit imports if it's < 100!